includes/Admin/Editor.php

```diff
@@ -340,6 +340,7 @@ public function get_editor_settings(): array {
 				'encodeMarkup'                 => $this->decoder->supports_decoding(),
 				'metaBoxes'                    => $this->meta_boxes->get_meta_boxes_per_location(),
 				'ffmpegCoreUrl'                => trailingslashit( WEBSTORIES_CDN_URL ) . 'js/@ffmpeg/core@0.10.0/dist/ffmpeg-core.js',
+				'localeData'                   => json_decode( load_script_textdomain( self::SCRIPT_HANDLE, 'web-stories' ), true ),
```

Comment:
Should this be moved to a trait or another class or something?

Reply:
Probably not worth it for this 1 line of code.